include/core/shape.h

```diff
@@ -931,7 +931,15 @@ class OpenCascadeShape : public Shape<dim>
     // distance tool with just the shell.
     if (shells.size() > 0)
       {
-        if (shells.size() > 1)
+        if (solids.size() == 1)
```

Comment:
The first `if` statement uses the object `solids` while the second `if` statement uses the object `shells`. I just want to make sure that this is normal :)

Reply:
Yeah clearly I would at least document this part.